client/src/components/EditProfile/EditComponents.js

```diff
@@ -0,0 +1,158 @@
+import styled from "styled-components";
+import Heading from "../Typography/Heading";
+import { mq } from "../../constants/theme";
+import SubmitButton from "../Button/SubmitButton";
+import {
+  ROYAL_BLUE,
+  LIGHTER_GRAY,
+  LIGHT_GRAY,
+  BLACK,
+  WHITE,
+} from "../../constants/colors";
+
```

Comment:
Importing the colors seems redundant when they are already included in the theme.